configs/templates/QuotaTemplate.proto

```diff
@@ -10,17 +10,8 @@ import "mixer/configs/mixer/TemplateExtensions.proto";
 option (istio.mixer.v1.config.template.template_variety) = TEMPLATE_VARIETY_REPORT;
 option (istio.mixer.v1.config.template.template_name) = "Quota";
 
-
-message Type{
-  map<string, istio.mixer.v1.config.descriptor.ValueType> dimensions = 2;
-
-  int64 max_amount = 5;
-  google.protobuf.Duration expiration = 6;
-}
-
-
 message Constructor{
```

Comment:
Same as above, the things in template.proto files should be called either `Type` or `Template` (not sure what we want to go with), and we'll generate the `Constructor` swapping the fields around so operators can provide expressions.